### ISSUE TITLE
UnitFrames fader does not respect correct distance to target

### ISSUE THREAD

**Issue by WiredUK** [tukui-org/ElvUI#1259]
### Troubleshooting Steps

Note the target unitframe is faded out despite me being 25-30 yards from the target. I can take a step closer and it will fade back in again. I have installed RangeDisplay addon to show the range, shown just above the target unitframe in the screenshot, hope that helps.

Not sure if relevant, but I have the new Phantom Reach priest talent to increase range by 15% so I am within range at ~45 feet. 

![image](https://github.com/user-attachments/assets/dbee8562-46c9-48fb-9f01-8bbaea7f959d)

### Describe the expected behavior and what actually happened?

When the range setting for fader on the target UnitFrame, the range is not accurate as to when it fades out. Note the distance doesn't seem to be consistent either. Very limited testing seems to indicate the physical size of the mob might play a factor, the larger the mob, the closer I need to be to unfade the frame, but that could be a red herring.

<img width="575" alt="image" src="https://github.com/user-attachments/assets/9743504d-0f3e-422a-a176-7c4d624dd63e">

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/main/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by kodewdle** [comment]
post talent page

**Comment by WiredUK** [comment]
@kodewdle This is my talent build:

`CIQAJSbRac/22NxZbHdYKOQzcMwYGDAAAAAAAAAAAAwMsNGzYWmNmZmZ2mZmF4B2YmZmxsxAjxMmFz2UzMYBmBAAAAAA`

![image](https://github.com/user-attachments/assets/738accfc-4904-479b-ba77-c87f3c5ddf08)

**Comment by HerrNirmas** [comment]
Same for Hunter Range seem 10m to short

**Comment by WiredUK** [comment]
@HerrNirmas Yes, same on my hunter too. Also confirmed on a warlock and a MoP Remix druid.

**Comment by WiredUK** [comment]
I think I have figured out the problem. In `ElvUI_Libraries/Core/LibRangeCheck-3.0/LibRangeCheck` on line 86 we have this line:
```
local GetSpellBookItemName = _G.GetSpellBookItemName or function(index, bookType)
```

If I change it to be this, the range check works:
```
local GetSpellBookItemName = function(index, bookType)
```

So it seems as if something else is setting up `_G.GetSpellBookItemName` to do something different. 

Also, the official repo for `LibRangeCheck` seems to have some [very different code](https://github.com/WeakAuras/LibRangeCheck-3.0/blob/main/LibRangeCheck-3.0/LibRangeCheck-3.0.lua) in there now, so perhaps this is as simple as updating to the latest version.

**Comment by kodewdle** [comment]
Thanks @WiredUK we actually have a fork of thiers, there was an update to theirs again (https://github.com/WeakAuras/LibRangeCheck-3.0/commit/7c733574977e7dfbb327a6c95c0af47b875b6bd4) which we accounted for (https://github.com/tukui-org/ElvUI/commit/ed65ca4f69b2cd657cc890136c4ded0e0d9f27fc) and fixed a bug in dev.  please retry and report back :)

dev version: [elvui-main.zip](https://api.tukui.org/v1/download/dev/elvui/main)

**Comment by WiredUK** [comment]
> dev version: [elvui-main.zip](https://api.tukui.org/v1/download/dev/elvui/main)

Works great, thanks @kodewdle

**Comment by kodewdle** [comment]
fixed in 13.73
